util/network/cniprovider/cni.go

```diff
@@ -62,40 +62,50 @@ func (c *cniProvider) initNetwork() error {
 		}
 		defer l.Unlock()
 	}
-	ns, err := c.New()
+	ns, err := c.New("test")
```

Comment:
I think this should be empty string and if empty string and implementation checks that it is set before calling `WithArgs`